src/main/java/com/group10/CI/ContinuousIntegrationServer.java

```diff
@@ -88,24 +133,27 @@ private Build handlePostRequest(HttpServletRequest request) throws IOException,
         testHandler.test();
 
         // if unable to execute test command, we do not want to send a notification
-        if(testHandler.getStatus() == Status.ERROR) return null;
+        if (testHandler.getStatus() == Status.ERROR)
+            return null;
 
         build.setTestStatus(testHandler.getStatus());
         System.out.println("Build and testing complete.");
 
         notifier.notifyGitHub(build);
 
-        if(!notifier.getSuccessfulDelivery()) System.out.println("Unable to notify Github.");
+        if (!notifier.getSuccessfulDelivery())
+            System.out.println("Unable to notify Github.");
         return build;
     }
 
-    private JSONObject getBody(HttpServletRequest request){
+    private JSONObject getBody(HttpServletRequest request) {
         StringBuilder stringBuilder = new StringBuilder();
 
         try {
             BufferedReader br = request.getReader();
             String line;
-            while ((line = br.readLine()) != null) stringBuilder.append(line);
+            while ((line = br.readLine()) != null)
+                stringBuilder.append(line);
             return new JSONObject(stringBuilder);
```

Comment:
I think that this line will result in an empty JSON body since the JSONObject() expects a string. If it's not working we can add the toString() method
```suggestion
            return new JSONObject(stringBuilder.toSting());
```

Reply:
This is fixed in main right now